lms/djangoapps/certificates/management/commands/resubmit_error_certificates.py

```diff
@@ -117,7 +116,7 @@ def _load_course_with_cache(self, course_key, course_cache):
         """Retrieve the course, then cache it to avoid Mongo queries. """
         course = (
             course_cache[course_key] if course_key in course_cache
-            else modulestore().get_course(course_key, depth=0)
+            else get_course_overview(course_key)
```

Comment:
Is _course_ a course or a course_overview? If the latter, is this change correct?

Reply:
The `course` will be a `CourseOverview` now. I think this is fine. It is mostly used to determine if a course exists before submitting a certificate for generation.

I thought this would be fine as the CourseOverview `get_course_overview` API function calls the model's `get_from_id` function which will:
- Attempt to retrieve the course_overview, but if that doesn't exist
- Attempt to retrieve the course from the modulestore, create a CourseOverview for it, and save it to the database

I will update the name so its a bit clearer what we are dealing with in this context.